utils/node-module-licenses.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+import csv, json, sys, os.path, re, io
+
+os.system("cd js/node && npx license-checker --relativeLicensePath --json > OUT")
+
+nv_re = re.compile(r'(@?[^@]*)@([^@]*)')
+git = "https://raw.githubusercontent.com/arangodb/arangodb/devel"
```

Comment:
Should we make this configurable?

Reply:
At least for current branch it makes sense to have `3.7` here. @Simran-B can you please confirm.

Reply:
This should be set to the respective branch name indeed (here: 3.7)